src/test/java/com/softserveinc/ita/pkuravskyi/test1.java

```diff
@@ -0,0 +1,44 @@
+package com.softserveinc.ita.pkuravskyi;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+import java.util.concurrent.TimeUnit;
+
+public class test1 {
+    private WebDriver driver;
+
+    @BeforeSuite
+    public void beforeSuite() {
+        //Installing chrome driver
+        System.setProperty("webdriver.chrome.driver", "E://SoftServe//chromedriver_win32//chromedriver.exe");
```

Comment:
place a chromedriver in this package 'main/java/com/softserveinc/ita' create here a package 'drivers'